### PR TITLE
Northwest_Class_4 _Bimbola_Lasisi- GitHomeworkFixErrors-Week_1

### PR DESCRIPTION
**Pull request by blasisi**
Answered the correct answers to the questions

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -44,70 +44,70 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-You send the changes of your local repository to Github
+Push changes  allow you to send the commit changes of your local repository to remote repository to Github
 
 ## Question 14
 
 What is Github?
 
-A place for us to store and share our code.
+A  git repository is hosting services,  help team member to work together on a project and stayed organized. Github used to share project , keep  record and store all code based files and keep local and remote copies of project, keep track of changes that has been pushed to local and remote repository .
```
Comment (myaskylar) [inline homework.md:90]: Nice definition!

File: homework.md [hunk 2]
```diff
@@ -4,31 +4,31 @@
...
 
-Purple
+white
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+13464
```
Comment (xtremelibasic) [inline homework.md:19]: Hi Bimbola, I think the answer here is 12240.

**Review by xtremelibasic — COMMENTED** [review]
Wonderful definitions!

File: homework.md (hunk 1 above)
Comment (blasisi) [reply homework.md:90]: > Nice definition!

Thank you so much Mya

**Pull request closed without merging**